lib/commands/project/project-command-base.ts

```diff
@@ -0,0 +1,12 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+export class ProjectCommandBase {
```

Comment:
why do we need this base class, can't we just add canExecute method of the command